WordPress/gutenberg
# Font family preview in the font family picker

## What?
Style font family option in the font family picker.

## Why?
To help users to select the desired font.
Fixes: https://github.com/WordPress/gutenberg/issues/51151

## How?
Using the [CustomSelectControl](https://wordpress.github.io/gutenberg/?path=/docs/components-customselectcontrol--docs) component and setting the right style.




## Testing Instructions
Open a font family picker and verify that each font family renders the family's name using the family's typography.


## Screenshot

| Before | After |
| --- | --- |
| ![image](https://github.com/user-attachments/assets/c16dc3f8-07bb-4e2d-8aff-5e0ee1b9b3b7) | ![image](https://github.com/user-attachments/assets/8e8c9e75-18ae-4bca-b728-5e3c0a1ead91) |
